core/ir/decodelib.c

```diff
@@ -128,7 +128,7 @@ get_thread_private_dcontext(void)
 void
```

Comment:
The commit title and description are not following the conventions in multiple respects:

+ The title should start with the issue number
+ The description should describe what is being fixed (yes, duplicating the title)
+ There should be a Fixes line at the bottom

Please see https://github.com/DynamoRIO/dynamorio/wiki/Code-Reviews#commit-messages